components/family-studies/api/src/main/java/org/phenotips/studies/family/Family.java

```diff
@@ -19,19 +19,25 @@
 
 import org.phenotips.Constants;
 import org.phenotips.data.Patient;
+import org.phenotips.studies.family.internal.Pedigree;
 
 import org.xwiki.model.EntityType;
 import org.xwiki.model.reference.DocumentReference;
 import org.xwiki.model.reference.EntityReference;
 
 import java.util.List;
+import java.util.Map;
+
+import com.xpn.xwiki.XWikiException;
+
+import net.sf.json.JSON;
 
 /**
  * @version $Id$
  */
 public interface Family
 {
-    /** The XClass used for storing family data. */
+    /** The class used for storing family data. */
```

Comment:
Leave this as `XClass`, it's a term used to refer to XWiki Classes, as opposed to Java classes.